src/main/java/emu/grasscutter/command/commands/TalentCommand.java

```diff
@@ -111,4 +117,78 @@ public void execute(Player sender, Player targetPlayer, List<String> args) {
             }
         }
     }
+
+    private void setAllAvatarTalentLevelMax(Player sender, Player player) {

```

Comment:
This seems very excessive for something that aims to be very similar to the n/e/q cases.